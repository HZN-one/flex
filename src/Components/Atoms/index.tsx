export * from './Button';
export * from './Icon';
export * from './FormInput';
export * from './TextField';