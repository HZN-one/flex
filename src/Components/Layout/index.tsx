export * from "./Initiator";
export * from "./FLSettingsNavbar";
