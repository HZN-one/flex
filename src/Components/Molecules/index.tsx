export * from "./FMAccordion";
export * from "./FMSearch";
export * from "./FMCard";
export * from "./FMStepper";
export * from "./FMTableSkeleton";
export * from "./FMTimeline";
export * from "./FMLabelValue";
export * from "./FMTabsMenu";
export * from "./FMInputAvatar";
