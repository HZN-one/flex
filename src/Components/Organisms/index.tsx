export * from "./FOCardWithImage";
export * from "./FOCardWithStepper";
export * from "./FOCardWithTextField";
export * from "./FODialogWithAlert";
export * from "./FOHeader";
export * from "./FOPaperRow";
export * from "./FOSideBar";
