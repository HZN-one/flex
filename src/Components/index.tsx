export * from './Atoms'
export * from './Layout'
