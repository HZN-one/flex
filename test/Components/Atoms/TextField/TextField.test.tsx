/**
 * @jest-environment jsdom
 */

 import * as React from 'react'
 import '@testing-library/jest-dom'

 
 import { TextFieldPegasus } from 'Components'
 
 import { render } from '@testing-library/react'
 
 describe('TextFieldPegasus', () => {
   it('should match snapshot', () => {
     const { container } = render(
       <TextFieldPegasus testID="input-example"  />
     )
     expect(container).toMatchSnapshot()
   })
   
 })
 