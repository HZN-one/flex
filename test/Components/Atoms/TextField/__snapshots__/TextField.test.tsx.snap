// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextFieldPegasus should match snapshot 1`] = `
<div>
  <div
    class="MuiFormControl-root css-1nrlq1o-MuiFormControl-root"
  >
    <label
      class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-standard MuiFormLabel-root MuiFormLabel-colorPrimary css-aqpgxn-MuiFormLabel-root-MuiInputLabel-root"
      data-shrink="false"
      for="component-error"
    />
    <div
      class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-1480iag-MuiInputBase-root-MuiInput-root"
      data-testid="input-example"
    >
      <input
        aria-describedby="component-error-text"
        aria-invalid="false"
        class="MuiInput-input MuiInputBase-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
        type="text"
        value=""
      />
    </div>
    <p
      class="MuiFormHelperText-root Mui-error MuiFormHelperText-sizeMedium css-1d1r5q-MuiFormHelperText-root"
      id="component-error-text"
    />
  </div>
</div>
`;
